polaris/polaris/models.py

```diff
@@ -99,6 +101,9 @@ class Transaction(models.Model):
         choices=MEMO_TYPES, default=MEMO_TYPES.text, max_length=10
     )
 
+    objects = models.Manager()
```

Comment:
what does this do?

Reply:
We already use it throughout the codebase, `Transaction.objects.filter(...)`. It's a builtin attribute for all django models, used for forming queries. I just defined it in the class so PyCharm would stop complaining.